hyperspy/component.py

```diff
@@ -470,6 +471,19 @@ def export(self, folder=None, name=None, format=None,
             self.as_signal(field='std').save(append2pathname(
                 filename, '_std'))
 
+    def default_traits_view(self):
```

Comment:
Nice solution. Could you just add some comments to the code to explain why you need to set the default view.